bridge-evm/docs/src/contracts/utils/ReceiptUtils.sol/library.ReceiptUtils.md

```
@@ -0,0 +1,107 @@
+Shortcut to convert receipt to hash
+
+*using [toEthSignedMessageHash](https://docs.openzeppelin.com/contracts/5.x/api/utils#MessageHashUtils-toEthSignedMessageHash-bytes32-) from OpenZeppelin's MessageHashUtils*
+
+
+```solidity
+function toHash(BridgeTypes.FullReceipt memory receipt)
+    internal
+    pure
+    returns (bytes32 hash);
+```
+**Parameters**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)